### DISCUSSION TITLE
Turnip driver support

### DISCUSSION THREAD

**Discussion by LorVC** [rafaelvcaetano/melonDS-android#1337] [Ideas]
Speed shouldn't be a problem nowadays, but still enabling support to change the GPU driver could increase drastically the performance at relatively the cost of implementing AdrenoTools

**Comment by Unknowncat1991** [comment]
This is useless because melonDS doesn't support vulkan as of the moment.

**Comment by WindowsNinetyTen** [comment]
Though I have called for the addition of Vulkan support.
